### PR TITLE
Update docs for workflows

### PR DESCRIPTION
**Pull request by ttung**
### Description
This encompasses the ingest, transform, and align workflows.

### Test plan
Visual inspection.

### CODE REVIEW

File: docs/backend/workflows.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+This workflow automatically invokes the [transform-gisaid workflow](#transform-gisaid) upon completion, using the [`RawGisaidDump`](https://github.com/chanzuckerberg/aspen/search?q=%22class+RawGisaidDump%22) as input.
+
+### Transform GISAID
+
+This workflow applies the [transform-gisaid script](https://github.com/nextstrain/ncov-ingest/blob/master/bin/transform-gisaid) to transform the data into an input format suitable for the nextstrain pipeline.  The input for this workflow is a [`RawGisaidDump`](https://github.com/chanzuckerberg/aspen/search?q=%22class+RawGisaidDump%22) object and records a [`ProcessedGisaidDump`](https://github.com/chanzuckerberg/aspen/search?q=%22class+ProcessedGisaidDump%22) object referencing the processed dump.  A [`GisaidDumpWorkflow`](https://github.com/chanzuckerberg/aspen/search?q=%22class+GisaidDumpWorkflow%22) is used to link the input to the output.
```
Comment (phoenixAja) [inline docs/backend/workflows.md]: can you expand a bit more on what suitable means here?

**Review by phoenixAja — APPROVED** [review]
awesome this is great!!! thanks for adding this, one last request is can you document the phylo workflow / how to kick it off (and what inputs/outputs it takes in/ creates)?

File: docs/backend/workflows.md (hunk 1 above)
Comment (ttung) [reply docs/backend/workflows.md]: I think it is just the metadata format.  I will clarify.

**Comment by ttung** [comment]
> awesome this is great!!! thanks for adding this, one last request is can you document the phylo workflow / how to kick it off (and what inputs/outputs it takes in/ creates)?

I just added it, pls comment if needed. :)

**Review by sidneymbell — APPROVED** [review]
This is super helpful, thank you Tony!

**Comment by phoenixAja** [comment]
> > awesome this is great!!! thanks for adding this, one last request is can you document the phylo workflow / how to kick it off (and what inputs/outputs it takes in/ creates)?
> 
> I just added it, pls comment if needed. :)

looks great!! 👍

**Pull request merged**